dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/icon/DefaultIconService.java

```diff
@@ -164,9 +164,22 @@ public void updateCustomIcon(String key, String description, String[] keywords)
       icon.setKeywords(keywords);
     }
 
+    icon.setCustom(isCustom);
```

Comment:
same as above, won't this always be true? 
afaik, a custom icon will always stay a custom icon.